hail/python/hail/vds/methods.py

```diff
@@ -39,6 +39,13 @@ def to_dense_mt(vds: 'VariantDataset') -> 'MatrixTable':
         Dataset in dense MatrixTable representation.
     """
     ref = vds.reference_data
+    # FIXME(chrisvittal) consider changing END semantics on VDS to make this better
+    # see https://github.com/hail-is/hail/issues/13183 for why this is here and more discussion
+    ref = ref.transmute_entries(END_GLOBAL=hl.locus(contig=ref.locus.contig,
```

Comment:
The function this calls is not super cheap, better to separate this out into the pieces I suggested on that issue

Reply:
Fine. I disagree though. This is simple, correct, and maintainable. It is still possible to densify past the end of a contig with your solution, `END` could be larger than the contig.

Reply:
if END is greater than the max position of a contig? That's bad data, I'm OK having wrong results for densify there. If you want to catch that, can do so without calling the locus constructor per entry anyhow I think